README.rst

```diff
@@ -5,6 +5,10 @@ BrotliCFFI
     :alt: Version
     :target: https://pypi.org/project/brotlicffi
 
+.. image:: https://anaconda.org/conda-forge/brotlicffi/badges/version.svg
```

Comment:
I prefer the Shields SVG here: https://img.shields.io/conda/vn/conda-forge/brotlicffi